packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/ssr.js

```diff
@@ -38,23 +37,23 @@ const {handler} = runtime.createHandler(options, (app) => {
     // Set HTTP security headers
     app.use(
         helmet({
+            // pwa-kit-runtime ensures that the Content-Security-Policy header always contains the
+            // directives required for PWA Kit to function. Add custom directives here.
             contentSecurityPolicy: {
-                useDefaults: true,
                 directives: {
-                    'img-src': ["'self'", '*.commercecloud.salesforce.com', 'data:'],
-                    'script-src': ["'self'", "'unsafe-eval'", 'storage.googleapis.com'],
-                    'connect-src': ["'self'", 'api.cquotient.com'],
-
-                    // Do not upgrade insecure requests for local development
-                    'upgrade-insecure-requests': isRemote() ? [] : null
+                    scriptSrc: [
+                        'storage.googleapis.com' // used by the service worker in /worker/main.js
```

Comment:
Why did we separate storage.googleapis.com from the other headers that were moved into pwa-kit-runtime?

Reply:
It's not a domain that we own, so we shouldn't implicitly trust it, plus it's good to keep the `hemet()` pattern as an example for users to copy from.

Previous discussion: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1457#discussion_r1358786062